kolibri/plugins/learn/assets/src/views/ExamPage/index.vue

```diff
@@ -456,6 +457,16 @@
         context:
           'Action that learner takes to submit their quiz answers so that the coach can review them.',
       },
+      jumpToQuestion: {
```

Comment:
**Learn > Home > Quiz**

* landmarks should be used sparingly, and I'm not sure I see a great utility for the ``section`` one here, especially since the ``aria-label`` does not provide much _information_ 🙂  
* ``ul`` element will not be recognized as a landmark, with or without the ``aria-label``. 
* ``aria-label``, not ``arialabel``

My suggestion was to wrap just the list of question in a div with ``role="navigation"`` and add the ``aria-label="Jump to question"``, could that be achieved?

![Ubuntu18 (start)  Running  - Oracle VM VirtualBox_017](https://user-images.githubusercontent.com/1457929/186923912-ee128385-e0cb-4bc2-97e1-3925205058b6.png)